dotCMS/src/main/java/com/dotcms/rest/api/v1/secret/ServiceIntegrationResource.java

```diff
@@ -111,7 +113,8 @@ public final Response listAvailableServices(@Context final HttpServletRequest re
     public final Response getServiceIntegrationByKey(
             @Context final HttpServletRequest request,
             @Context final HttpServletResponse response,
-            @PathParam("key") final String serviceKey
+            @PathParam("key") final String serviceKey,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "key" appears 4 times in this file; the first occurrence is on line 116](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4960082)